crates/router/src/core/payments/helpers.rs

```diff
@@ -2077,7 +2077,14 @@ pub async fn get_merchant_connector_account(
                 )?;
 
             #[cfg(feature = "kms")]
-            let private_key = state.kms_secrets.jwekey.peek().tunnel_private_key.clone();
```

Comment:
why are we making this change is it needed?

Reply:
here again,we need to use the kms key without the option value...hence the change and ok_or